app/scripts/views/mixins/upgrade-session-mixin.js

```diff
@@ -95,6 +100,7 @@ define(function (require, exports, module) {
           emailSent: this.model.get('emailSent'),
           gatedHref: options.gatedHref,
           isPanelOpen: this.isPanelOpen(),
+          lastCheckedTime: this._getLastCheckedTimeString(),
```

Comment:
You can add a `setInitialContext` method inside of the mixin that sets only `lastCheckedTime` so that you don't have to add this field to every view that uses it.